src/MAUI/Maui.Samples/Samples/GraphicsOverlay/SketchOnMap/SketchOnMap.xaml.cs

```diff
@@ -189,12 +188,17 @@ private async void EditButtonClick(object sender, EventArgs e)
         {
             try
             {
+                // Change the background of the currently selected tool from gray to red.
```

Comment:
Suggestion: we could potentially combine these comments to say something like "Update UI." as we are already quite comment heavy and I'm not sure we are adding much value by being this descriptive to the user.

Reply:
Agreed, especially since `SelectTool` already denotes that we're changing the color of the button background.